LayoutDemo/app/src/main/java/edu/h2/layoutdemo/login/presenter/LoginPresenterImplement.java

```diff
@@ -53,15 +82,22 @@ public void onComplete() {
     }
 
     /**
-     * verify information between driver get from server by driverId and from screen
-     * @param driver
+     *
+     * @param isLogin
+     * @param params
      */
-    public void onLogin(Driver driver, DriverAuthenticateUseCase.Params params) {
-        if (driver.getBusId().equals(params.busId) && driver.getPassword().equals(params.password)){
+    public void onLogin(Boolean isLogin, DriverAuthenticateUseCase.Params params, int loginCount) {
```

Comment:
Remove unnecessary parameter `loginCount`. The method can be tested by calling onLogin many times